pages/portfolio.html

```diff
@@ -4,9 +4,90 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio</title>
+  <link href="/styles/style.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/3e8fd03e4e.js"
+  crossorigin="anonymous"></script>
 </head>
 <body>
+<header>
+  <nav>
+    <ul class="top-bottom-links">
+      <li>Portfolio</li>
+      <li><a href="/pages/index.html">Home</a></li>
+      <li><a href="/pages/about.html">About</a></li>
+    </ul>
+  </nav>
+</header>
+<h2>Selected Projects</h2>
+<p>
+<ul>
+  <a href="https://github.com/DynamicDev3000/task-list-api">One-to-many relationships API</a>
+  <ul>
+    <li>Working with the Flask package</li>
+    <li>Creating models</li>
+    <li>Creating conventional RESTful CRUD routes for a model</li>
+    <li>Reading query parameters to create custom behavior</li>
+    <li>Create unconventional routes for custom behavior</li>
+    <li>Apply knowledge about making requests in Python, to call an API inside of an API</li>
+    <li>Apply knowledge about environment variables</li>
+    <li>Creating a one-to-many relationship between two models</li>
+</ul>
+</ul>
+</p>
```

Comment:
The ul tag on line 24 implicitly closes the p tag on line 23. Lists cannot be placed within paragraphs, as they start their own block section. As a result, the closing p tag here ends up being an unbalanced closing tag.

And as I'm sure you can guess by now, running your html through a validator would catch this.